plaidml/bridge/tensorflow/tests/utils.cc

```diff
@@ -0,0 +1,55 @@
+#include "plaidml/bridge/tensorflow/tests/utils.h"
```

Comment:
Could we merge all this into `codegen_test.cc`? Right now we have a `py_library` called `util` and a `cc_library` called `utils`.